src/pages/LogView/LoadingPage/useResolveLogURLAndRenderingType.ts

```diff
@@ -189,17 +201,32 @@ export const useResolveLogURL = ({
           text: false,
         });
       downloadURL = rawLogURL;
+      if ((renderingTypeFromQuery || "") in LogRenderingTypes) {
+        renderingType = renderingTypeFromQuery as LogRenderingTypes;
+      } else if (!renderingTypeFromQuery) {
+        renderingType = LogRenderingTypes.Default;
+      } else {
+        renderingType = LogRenderingTypes.Default;
```

Comment:
I think this can be refactored out. The first if statement handles the case where renderingTypeFromQuery is undefined. `renderingTypeFromQuery` can't be in `LogRenderingTypes`

Reply:
If we assign a default value for `renderingType` like my suggestion on on L115 this assignment in the else statement becomes unnecessary as well

Reply:
I may be missing what you mean but Afaict `renderingType` doesn't get overridden anywhere else so I still think it's not necessary. Can you point me to where it does?